lib/teslamate_web/live/charge_live/cost.html.heex

```diff
@@ -1,7 +1,7 @@
 <nav class="breadcrumb" aria-label="breadcrumbs">
   <ul>
-    <li><%= live_redirect gettext("Home"), to: "/" %></li>
-    <li class="is-active"><%= live_redirect gettext("Charge Cost"), to: "" %></li>
+    <li><%= live_redirect(gettext("Home"), to: "/") %></li>
+    <li class="is-active"><%= live_redirect(gettext("Charge Cost"), to: "") %></li>
```

Comment:
These changes (adding parens) seems like a personal preference so I'd revert them since they don't add functionality and it looks like Adrian has consistently chosen to omit them.

Reply:
skipped the Phoenix update for now - not experienced enough in Elixir to do that...